saos-webapp/src/test/java/pl/edu/icm/saos/webapp/court/TestCourtsFactory.java

```diff
@@ -17,15 +17,26 @@
 	
 	//------------------------ LOGIC --------------------------
 	
-	public List<SimpleEntity> getSimpleEntities() {
-		SimpleEntity divisionOne = new SimpleEntity();
+	public List<SimpleEntity> createSimpleEntities() {
+		SimpleEntity entityOne = new SimpleEntity();
 		SimpleEntity divisionTwo = new SimpleEntity();
```

Comment:
I looks strange when one object is named entityOne and the other one divisionTwo.